### PR TITLE
EUI-6921 Fix old case ref issues

### PR DESCRIPTION
**Pull request by DavidJayakumar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-6921

### Change description ###

Fix to display the correct case referecne from the route

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/shared/components/case-viewer/case-specific-access-success/case-specific-access-success.component.spec.ts [hunk 1]
```diff
@@ -35,6 +33,7 @@ describe('CaseSpecificAccessSuccessComponent', () => {
   it('should create component and show the correct Case Reference', () => {
     expect(component).toBeDefined();
     const confirmationMessageElement = fixture.debugElement.nativeElement.querySelector('.govuk-panel__body');
+    console.log('test' , JSON.stringify(confirmationMessageElement.textContent));
```
Comment (udaydenduluri33) [inline src/shared/components/case-viewer/case-specific-access-success/case-specific-access-success.component.spec.ts:36]: Please remove console.log

Comment (connorpgpmcelroy) [inline src/shared/components/case-viewer/case-specific-access-success/case-specific-access-success.component.spec.ts:36]: console.log in here

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "4.18.19-specific-access-reason-merged",
+  "version": "4.18.19-Fix-old-case-id",
```
Comment (connorpgpmcelroy) [inline package.json:3]: very minor comment - prefer all lower case letters as stops confusion when people use the tag

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Remove console.log

**Review by udaydenduluri33 — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]

**Pull request merged**